File: 02_ProgrammingFundamentals/_01_BasicSyntaxConditionalStatementsAndLoops/_01_Lecture/_02_Passed.java
package _02_ProgrammingFundamentals._01_BasicSyntaxConditionalStatementsAndLoops._01_Lecture;

import java.util.Scanner;

public class _02_Passed {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        double grade = Double.parseDouble(scan.nextLine());

        // Print result
        if (grade >= 3.00) {
            System.out.println("Passed!");
        }

        scan.close();
    }
}

File: 24_C#Fundamentals/_01_BasicSyntaxConditionalStatementsAndLoops/_01_Lecture/_02_Passed/_02_Passed.cs
using System;

public class _02_Passed
{
    public static void Main(string[] args)
    {
        // Read input value
        double grade = double.Parse(Console.ReadLine());

        // Print result
        if (grade >= 3.00)
        {
            Console.WriteLine("Passed!");
        }
    }
}
